### PR TITLE
Media queries Added in this branch

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
Media queries are added in this branch. The design is responsive. Images are updated.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -338,3 +338,129 @@ ul {
...
+  .card-desktop-img {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  * {
+    box-sizing: border-box;
+  }
+
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that you are using a media query for your desktop version of 768px or wider. This is a project requirement. :eyes: :eyes: 
image

File: style.css [hunk 2]
```diff
@@ -338,3 +338,129 @@ ul {
...
+    display: none;
+  }
+
+  .card-img {
+    display: none;
+  }
+
+  /* .main2 {
+    list-style-type: none;
+  } */
```
Comment (Santiago220991) [inline style.css]: Please remove commented code. It makes your project looks messy. :+1: :smiley: :smiley:

File: style.css [hunk 3]
```diff
@@ -338,3 +338,129 @@ ul {
...
+
+  .navbar {
+    justify-content: space-between;
+    padding: 48px 130px 48px 130px;
+  }
```
Comment (Santiago220991) [inline style.css:385]: Kindly make sure that your project sticks as much as possible to the Figma template provided. Please change the color of the navbar links with the color provided in the Figma template. :pray: :pray: 
Your design
image

Figma design
image

File: style.css [hunk 4]
```diff
@@ -338,3 +338,129 @@ ul {
...
+    flex-direction: column;
+    background-image: url(./images/header2x.png);
+    width: 100%;
+    height: auto;
+  }
+
+  .content1 {
+    padding: 30px;
+    margin: 315px 260px;
+  }
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that you are using the same color text provided in the Figma template. Also please make sure that you are using the same font color.
Your design
image

Figma design
image

File: style.css [hunk 5]
```diff
@@ -338,3 +338,129 @@ ul {
...
+    justify-content: space-between;
+    padding: 48px 130px 48px 130px;
+  }
+
+  .section-card {
+    margin: 112px 215px 36px 215px;
+    width: 1156px;
+    height: 496px;
+    grid-template-columns: 50% 50%;
+  }
```
Comment (Santiago220991) [inline style.css]: - Kindly make sure that your project shows all the content without needing horizontal scrolling. :eyes: :eyes: Also please center these cards there is just a left margin but not a right margin.

Your design
image

Figma design
image

File: style.css [hunk 6]
```diff
@@ -338,3 +338,129 @@ ul {
...
+    width: 100%;
+    background-image: url(./images/contact-form2x.png);
+    background-size: 92%;
+    background-position: 2em 12em;
+  }
+
+  #contact-text {
+    line-height: 10px;
+    margin-left: 87px;
+  }
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that your design sticks as much as possible to the Figma template provided.
- Please center the contact text. :eyes: :eyes: 
image

- Kindly make sure that the inputs have the same width that the provided in the Figma template. :+1: :smiley: :smiley: 
Your design
image

Figma design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Rana-Mudassir, 
I'm Santiago your reviewer for this PR. :wave: :wave: 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Rana-Mudassir 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**